### ISSUE TITLE
amplify remove function --help is incorrect

### ISSUE THREAD

**Issue by stuartlangridge** [aws-amplify/amplify-cli#12101]
### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

v18.14.1

### Amplify CLI Version

10.7.3

### What operating system are you using?

Ubuntu

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

No manual changes made

### Describe the bug

The help for amplify function remove is not correct.

```
$ npx amplify remove function -h
USAGE
  amplify function remove

DESCRIPTION
  Builds all the functions in the project

LEARN MORE
  https://docs.amplify.aws/cli/commands/function
```

Remove doesn't build all the functions, and the LEARN MORE link is 404.

### Expected behavior

The help should be correct (in particular, can I do `amplify remove function someFunctionName`?

The help link should point at documentation (I don't know where actual CLI documentation is; https://docs.amplify.aws/cli/function/ has some details but it's not a man page or similar).

### Reproduction steps

 amplify remove function -h

### Log output

<details>

```
# Put your logs below this line


```

</details>

### Before submitting, please confirm:

- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

**Comment by ykethan** [comment]
related to fix on https://github.com/aws-amplify/amplify-cli/pull/12037

**Comment by edwardfoyle** [comment]
This is resolved in version 10.8.0 of the CLI. Please upgrade to try it out.
